lib/web_ui/lib/src/engine/canvas_pool.dart

```diff
@@ -145,14 +145,17 @@ class _CanvasPool extends _SaveStackTracking {
     _replayClipStack();
   }
 
+  static bool _fontSetTested = false;
```

Comment:
It's not immediately clear what role this field plays. Let's leave a comment. In particular it is good to know that it's for Firefox only.

Reply:
Removed optimization from this PR, will address later.